digitalocean/loadbalancer/resource_loadbalancer.go

```diff
@@ -475,6 +475,13 @@ func resourceDigitalOceanLoadBalancerV0() *schema.Resource {
 							ValidateFunc: validation.NoZeroValues,
 							Description:  "name of certificate required for TLS handshaking",
 						},
+						"certificate_id": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Computed:     true,
+							ValidateFunc: validation.NoZeroValues,
```

Comment:
If we do add this, we can remove `Optional` and `ValidateFunc` as it is read-only.

Reply:
done!